packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/tags/DeleteTagsTests.java

```diff
@@ -0,0 +1,61 @@
+package org.alfresco.rest.tags;
```

Comment:
FWIW we usually put the copyright header at the top of files (even test classes) in alfresco-community-repo and alfresco-enterprise-repo. For some reason we don't have headers in the tests in acs-packaging.